docs/img/ErrorSequence.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2020-08-18T11:37:41.233Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.125 Safari/537.36" etag="LQhDJ6Cje1SLURy3rB0D" version="13.4.9" type="device"><diagram id="kgpKYQtTHZ0yAKxKKP6v" name="Page-1">5VrbUuM4EP2aVO0+kPI1l0cSLkMVM1DLVgH7pthyrEWRjCTnwtevJMvxNSQQzDIzqakhalkdu0/r9GklPXe6WF8ykMTfaQhxz7HCdc896zmO7TlOT/2zwk1mGdpuZpgzFJqLCsMdeoHGaBlrikLIKxcKSrFASdUYUEJgICo2wBhdVS+LKK5+agLmsGG4CwBuWu9RKOLMOvKtwv4Nonmcf7JtmZkFyC82Bh6DkK5KJve8504ZpSJ7t1hPIVbBy+OSrbvYMbu9MQaJOGSBS9LJxZqNg821lZCb+78dcHniZ16WAKfmgb+lM3O/YpMHQd56ot6mC3yNIogRkaNJAhlaQAGZnMHGfFvYJhISAaRNzdt6jDFIOJppt5a0MBikjKMl/AvyDHltpSkJYWhWbcOmprhg9GkLhJpuRsEEZgmZgOuSyUTlElJ5g2wjLzGzTg6mSdGRGa4KvLeoxiWs3dwITI7Nt64LGOQbg8QbUBk0UGlAklBEhP5cf9Lzz2pwUCZiOqcE4DIgRWCt9wT21QQ6ONqevz/YzVg7ncV61Ij1+VI9smP9oAJFKAACUdKIv3pgOYdPMZoTaZpRIehCxg2S8FTxjrJhGjwdkcIwrHBTM6QMYnl3yypVtUXILL1VSVNA4VpVLOyRW3XBacoCaFaVqaXmyPO9/h5XArA5FA1XGrLtEx2EYnj1aOGHfyZ3VwE/i4Pvt6fj1YndQPEezmJKn6TxNEl6jtxSrq3i/wdPZzxgaAbZn78Z0Xn2fqIb+R0RXStszpckulcz7GcgOodfLdePL+sfY+slAtEc4Xv3JBde5WD7E74hAZR8xVRsOye5GqMdGvadMT6x+t7QMPiRZHhi19jQ8Uc5qb2VD+2ht8/VDj4sXOUX0iji8FjObE+IpvjbmQHXYAbxLeVIF8NSJtRTRNBEWmOxwAWtaUpdrOeqQ+jPAEdBf92gv0iSqJH/mqcjhPGUYsr0jbgXF5Z8bROpsQ2PTqWaCLS95n51WvdrR9TY1CVamVtEqxLZEkmcTCVDZK7uIEmwUSt8Z2GTq1VFW8VIwLsEBMq2krhUMZuB4Gmu4blJRVYEtT0E7OlGrkJCRcnqyx1ThxGYPMAwEgp9g+jwYyrYwK2h5NtNVm2DyRt3VcG8Nlb1LPl51pSSSEIiOqbW8s6xtQf5YA9mUg8eNVqe6+eGs7Xxlo025VGpkGbGNRIPGdyjgRlXHMpx4U8NNqVB3VsIeLxNlp0Z0bXy9Xy/P/LHxatK1+6gP7BKL+d9ZaCur/f5/TiN3M73OxgF8YJWQvmWRuqesjMVuS/VA5X0cxADIsGRy4i6lsB8hYhh9cqt0A73sVOHm+EA3dFNitlOVQPYg/H7ssgZu/Xmqu6q48RpEY6fpRMiLPlir1Y4SBO8Xlm2VSOPsX1Y/fdeSYyj5HqzrsiOlss9oMTgL9XTHoWb7/2Ph3etwB2gqj+pp90T2N1p99P0tF/l8G5/fZH1UZP0vsT5jFbXtQZVXWLXXBxamvyR9bqjL9DljpvbcTgpcWdv6vROLePlNFMwz6n6UmbyPF66/6azMHwWA5fYxUTPmbZ54AKIlNe96OwvVhbLhp/bFXQn42s93/sll0krqz9ynOGHpLo/sl8R+o0i8AZlX9NkDVdfIPVzLv6lhdvAqnHQ8DDh5nZ2ztrss65kH8VkPa/0VHkjVQNEz1YDrPd2fhZGqNZv5eMxY8q3YQBJJhnq0C1QGOJdxz9VfdEFULVvH7Y7pgSU19EJWztQTgOom3o7W21bs1Y5JUCpYPVDhJ7+RQGWNFxGFpBQMxdPKAm5fkL5gFlPnZ0MFQUhyM+ISiXiy57dHYe/WxcLLeqxbacOO0uAZo91t+tINQd/AUKdFyv53DpLgEI8OwZpuZymIk8o9U1LX/6dYpRJ1AAQTQOU8m02MZlNCkGtMLJDGRXClMsNLf2o/9AiwXAhXeQfxBMYKK37uySO6x6YOPY7OF4Oi5/kZIW4+GGTe/4f</diagram></mxfile>
```

Comment:
This isn't specific to the diagram, but it made me wonder (which wouldn't have happened if not for the diagrams so good job 👍), if say the first subscriber has an error which would generate a sync error, does the Hub not send the original notification to the rest of the subscribers? In the case where the subscriber's http response shows an error maybe this would be possible, but if they respond with a 202 then send the Hub a sync error after, then there would be no way for the Hub to delay meaning all subscribers would receive the original notification. That feels fine to me, as long as the Hub additionally sends the sync error, but I think both cases should be consistent. Either we send the notification and a sync error or we don't send the notification (I am leaning towards the former).

Reply:
Actually I think I misread the diagram. The event notifications are sent to both subscribers then one causes the error which generates the sync error to the other. That looks good to me.

The lanes are a bit off though, the box for the webhook app doesn't line up with the dotted line.

Reply:
Realigned the box and dotted line